api/src/test/java/org/openmrs/module/openconceptlab/ImportServiceTest.java

```diff
@@ -302,7 +303,7 @@ public void update_shouldUpdateReleaseVersion() throws Exception {
     }
 
 	@Ignore("This test passes locally, but fails on Travis-CI")
-	@NotTransactional
+	@Transactional(propagation = Propagation.NEVER)

```

Comment:
All of the tests that use this annotation are @Ignored so I guess I don't know if this is the right fix.  I'm just noticing that now.  I guess we should remove the @Ignore annotation and see what happens with these.  Maybe Github Actions will pass where Travis-CI did not...